src/sentry/api/base.py

```diff
@@ -149,19 +150,22 @@ def dispatch(self, request, *args, **kwargs):
         Identical to rest framework's dispatch except we add the ability
         to convert arguments (for common URL params).
         """
-        self.args = args
-        self.kwargs = kwargs
-        request = self.initialize_request(request, *args, **kwargs)
-        self.load_json_body(request)
-        self.request = request
-        self.headers = self.default_response_headers  # deprecate?
+        with sentry_sdk.start_span(op="PERF: base.dispatch - setup"):
```

Comment:
one thing would be to have consistent naming scheme for these spans.  Are we going to leave this in or are we going to take them out after we are done looking into this endpoint?

Reply:
I'm planning to take out unnecessary spans and keeping the spans that are ultimately useful on production.